roles/lib_openshift/src/class/oc_project.py

```diff
@@ -0,0 +1,183 @@
+# pylint: skip-file
+# flake8: noqa
+
+
+# pylint: disable=too-many-instance-attributes
+class OCProject(OpenShiftCLI):
+    ''' Class to wrap the oc command line tools '''
+    kind = 'namespace'
```

Comment:
When modifying the project's properties, oc does not allow certain edits when modifying through 'project' but when going through 'namespace' it does allow it.  That's why we chose namespace here.